index.html

```diff
@@ -1,50 +1,221 @@
-    <title>Portfolio</title>
+   <meta charset="UTF-8">
+   <meta name="viewport" content="width=device-width, initial-scale=1.0">
+   <link rel="stylesheet" href="styles.css">
+   <title>Portfolio</title>
 </head>
+
 <body>
-    <header class="toolbar">
+   <header class="toolbar">
       <a href="#">
-      <img src="./images/my-logo.svg" alt="my logo" class="my-logo">
+         <img src="./images/my-logo.svg" alt="my logo" class="my-logo">
       </a>
       <nav>
          <a href="#">
-            <img 
-            src="./images/hamburger.svg" 
-            alt="hamburger menu"
-            class="hamburger-img" id="hamburger">
-            </a>
+            <img src="./images/hamburger.svg" alt="hamburger menu" class="hamburger-img" id="hamburger">
+         </a>
       </nav>
-      </header>
-         <section class="main">
-            <div>
-               <h1 class="headline">I'm Ezinwa charles <br>Glad to see you!</h1>
-               <p class="desc">
-                  I’m a software developer! I can help you build a product , feature or website Look through some of my work and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
-               </p>
-               <p class="connect-msg">LET'S CONNECT</p>
-            </div>
-              <ul class="social-icons">
-                 <li><a href="https://twitter.com/ezinwacharles?t=szrnkroJTcBoT-GxDzBZOg&s=09">
-                  <img src="./images/twitter.svg" alt="twitter handle">
+   </header>
+   <section class="main">
+      <div>
+         <h1 class="headline">I'm Ezinwa charles <br>Glad to see you!</h1>
+         <p class="desc">
+            I’m a software developer! I can help you build a product , feature or website Look through some of my work
+            and experience! If you like what you see and have a project you need coded, don’t hestiate to contact me.
+         </p>
+         <p class="connect-msg">LET'S CONNECT</p>
+      </div>
+      <ul class="social-icons">
+         <li><a href="https://twitter.com/ezinwacharles?t=szrnkroJTcBoT-GxDzBZOg&s=09">
+               <img src="./images/twitter.svg" alt="twitter handle">
+            </a></li>
+         <li><a href="https://www.linkedin.com/in/ezinwa-charles-4401aa28/">
+               <img src="./images/linkedin.svg" alt="linkedin handle">
+            </a></li>
+         <li><a href="https://web.facebook.com/ezinwa.charles">
+               <img src="./images/media.svg" alt="another social handle">
+            </a></li>
+         <li><a href="https://github.com/cloud007bond">
+               <img src="./images/github.svg" alt="github handle">
+            </a></li>
+         <li> <a href="https://www.instagram.com/ichbindeutschen/">
+               <img src="./images/social.svg" alt="social handle">
+            </a></li>
+      </ul>
+   </section>
+
+   <!-- Here begins the works section -->
+   <section class="works-wrapper">
```

Comment:
[OPTIONAL]
- [ ] It would have been more beneficial for your practice if you incorporated the `css grid` with your `flexbox` to layout your section, kindly give it a try sometime for practice.